addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -97,7 +105,35 @@ def _l10n_it_edi_check_invoice_configuration(self, invoice):
 
         # <1.4.1>
         if not buyer.vat and not buyer.l10n_it_codice_fiscale and buyer.country_id.code == 'IT':
-            errors.append(_("The buyer, %s, or his company must have either a VAT number either a tax code (Codice Fiscale).", buyer.display_name))
+            errors.append(_("The buyer, %s, or his company must have a VAT number and/or a tax code (Codice Fiscale).", buyer.display_name))
+
+        # <2.2.1>
+        for invoice_line in invoice.invoice_line_ids:
+            if not invoice_line.display_type and len(invoice_line.tax_ids) != 1:
+                raise UserError(_("You must select one and only one tax by line."))
+
+        for tax_line in invoice.line_ids.filtered(lambda line: line.tax_line_id):
+            if not tax_line.tax_line_id.l10n_it_kind_exoneration and tax_line.tax_line_id.amount == 0:
+                errors.append(_("%s has an amount of 0.0, you must indicate the kind of exoneration.", tax_line.name))
+
+        return errors
+
+    def _l10n_it_edi_check_simplified_invoice_configuration(self, invoice):
+        errors = self._l10n_it_edi_check_buyer_invoice_configuration(invoice)
+        template = self.env.ref('l10n_it_edi.account_invoice_it_simplified_FatturaPA_export', raise_if_not_found=False)
+
+        if errors and template:
+            buyer = invoice.commercial_partner_id
+            if ((not buyer.country_id or buyer.country_id.code == 'IT') and
+                    (buyer.l10n_it_codice_fiscale or buyer.vat[:2].upper() == 'IT' or buyer.vat[:2].isdecimal()) and
```

Comment:
this creates a traceback when the customer has no details defined (no vat, no codice fiscale).
![image](https://user-images.githubusercontent.com/46533567/166949999-c20a2fc1-a949-40dc-b5f9-bd4d4ab412eb.png)

Maybe something like this?
```python3
if (
        (not buyer.country_id or buyer.country_id.code == 'IT')
        and buyer.l10n_it_codice_fiscale or (buyer.vat and (buyer.vat[:2].upper() == 'IT' or buyer.vat[:2].isdecimal()))
        and invoice.amount_total <= 400
):
    return []
```

Reply:
True but there are some missing brackets, more like

```suggestion
            if ((not buyer.country_id or buyer.country_id.code == 'IT')
                and (buyer.l10n_it_codice_fiscale or (buyer.vat and (buyer.vat[:2].upper() == 'IT' or buyer.vat[:2].isdecimal())))
                and invoice.amount_total <= 400):
                return []
```